cookieConsent/index.js

```diff
@@ -0,0 +1,7 @@
+import * as OneTrust from "./OneTrust";
+
+const cookieConsent = {
+  OneTrust: OneTrust.default,
+};
+
+export { cookieConsent };
```

Comment:
This should ideally be a factory that takes in source config and returns an integrations object which will be merged with integration object in the `load` call

Reply:
I would go one step further and write the merge logic within the cookie consent module itself.